src/tools.cpp

```diff
@@ -725,6 +725,20 @@ std::string getCombatName(CombatType_t combatType)
 	return "unknown";
 }
 
+CombatType_t getCombatTypeByName(const std::string& name)
+{
+	auto it = std::find_if(combatTypeNames.begin(), combatTypeNames.end(),
+		[&](std::pair<CombatType_t, std::string> const& pair)
+	{
```

Comment:
One minor thing: please check if this is consistent with lambda formatting elsewhere. IIRC braces in lambda expressions should be attached to the first line of the expression.

Reply:
It's working fine, I didn't see any problem with this modification. I update with your review, compiled and yeah, all right.

Reply:
I was referring to putting `{` in the previous line, because that is, I think how it's formatted elsewhere in the project.

Reply:
Oooh, sorry, haha. I guess so, I'm still adapting to the project, Ty.